eahub/config/urls.py

```diff
@@ -47,6 +47,7 @@
     path('user/<int:legacy_record>/', views.LegacyRedirectView.as_view(pattern_name='profile_legacy')),
     path('user/<slug:slug>/', views.LegacyRedirectView.as_view(pattern_name='profile')),
     path('user/profiles/', views.LegacyRedirectView.as_view(pattern_name='profiles')),
+    path('user/profiles/offers', views.LegacyRedirectView.as_view(pattern_name='profiles')),
```

Comment:
Trailing slash needed.

Reply:
> Do you happen to know what exactly was at this URL on the old Hub?

It wasn't cached or documented anywhere. I doubt that it was in regular use anyway. My opinion is that spending more time on figuring this out isn't worth it.